src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -125,8 +125,8 @@ public EventSource(Configuration configuration)
                 }
                 catch (Exception e)
                 {
-                    _logger.LogError("Encountered an error connecting to EventSource: {0}", e.Message);
-                    _logger.LogDebug(e.ToString());
+                    _logger.LogError( 0, e, "Encountered an error connecting to EventSource: {0}", e.Message);
```

Comment:
The exception is really important to pass through.  It contains the details of what actually happened.

When you're using async/await, and an AggregateException is thrown, you end up with this log message:

```Encountered an error connecting to EventSource: One or more errors occurred.```

By not passing along the exception, we have no way to know exactly what's going on.